sky/packages/sky/lib/base/lerp.dart

```diff
@@ -4,6 +4,7 @@
 
 import 'dart:sky';
```

Comment:
Well, "num.lerp" is gonna require help from the dart team. So this might be more involved that it might appear.